includes/phing/src/Tasks/PlatformVersionsTask.php

```diff
@@ -139,6 +139,13 @@ public function main()
         $this->getProject()->setNewProperty('platform.package.version.latest', $latest_version);
         $this->getProject()->setNewProperty('platform.package.version.current', $this->versionprop);
 
+        if ($this->_prerelease == true) {
```

Comment:
I think this is the configuration that you are checking. So if property `platform.package.download.prerelease = false` then you will always display this message, regardless if you are using an official release or a pre release.

If you want to keep this logic I would suggest changing the message to something like: `Please be aware that you have enabled the downloading of pre-releases. Pre-releases are never deployed to production. So this feature should only be used for testing purposes.`

Reply:
Done